stl/inc/variant

```diff
@@ -371,11 +353,9 @@ struct _Meta_cartesian_product_<_List1<_List2<_Items...>, _Lists...>> {
 
 #define _STL_STAMP(n, x) x(_STL_STAMP##n, n)
 
-// VARIANT OF VALUE TYPES [variant.variant]
 template <class... _Types>
 class variant;
 
-// VARIANT HELPER CLASSES [variant.helper]

```

Comment:
These stable name citations are being dropped (also below). @CaseyCarter should they be preserved by themselves?

Here is the full list of stable names being dropped:

```cpp
// STRUCT nullopt_t [optional.nullopt]
// CLASS bad_optional_access [optional.bad_optional_access]
// CLASS TEMPLATE optional [optional.object]
// COMPARISONS WITH nullopt [optional.nullops]
// COMPARISONS WITH T [optional.comp_with_t]
// FUNCTION TEMPLATE swap [optional.specalg]
// FUNCTION TEMPLATE make_optional [optional.specalg]
// STRUCT TEMPLATE SPECIALIZATION hash [optional.hash]
// VARIANT OF VALUE TYPES [variant.variant]
// VARIANT HELPER CLASSES [variant.helper]
// CLASS bad_variant_access [variant.bad.access]
// CLASS monostate [variant.monostate]
```